### ISSUE TITLE
Horizon Chase 2 not CTD

### ISSUE THREAD

**Issue by ghallok** [Ryujinx/Ryujinx-Games-List#4743]
### Game Name

horizon chase 2

### Game Update Version

1.41

### Emulator Build Version

1.1.104

### Graphics Backend

OpenGL

### Hardware Specs

- ##### CPU: *(e.g. i7-6700)*
- ##### GPU: *(e.g. NVIDIA RTX 2070)*
- ##### RAM: *(e.g. 16GB)*
ryzen 3 5300g
gtx 1050ti 4gb
16gb ddr4

### Game State

Game CTD immediately after the epic games screen both on vulkan and opengl

**Comment by gdkchan** [comment]
Log is missing.

**Comment by ghallok** [comment]
I got it to play but it plays horrible, and only worked in handheld mode so  log doesn't reflect ctd now. Still won't play on yuzu either, not sure where that log is.

**Comment by ghallok** [comment]
Put the ryujinx log on but again, i got it to play but fps was all over the place. Not sure what's happening, this is the only game i've encountered that issue in.

**Comment by gdkchan** [comment]
FWIW I tried it here. It's working but does have stability issues (crashed and froze once).
The race itself was full speed the entire time though.
![image](https://github.com/Ryujinx/Ryujinx-Games-List/assets/5624669/3ac0898b-956a-417d-80d8-43a4b20eee8b)

**Comment by ghallok** [comment]
Yeah hopefully it gets fixed, i was really looking forward to playing it. I did 4 races but the fps drops were really bad and it's just not playable right now.

**Comment by Tvangeste** [comment]
For me the game just hangs after the Epic logo, I see a black screen at 30 fps and nothing happens. Tried both Vulkan and OGL, and all possible combinations of settings, nothing helps.

**Comment by CWRules** [comment]
Seeing the same issue as Tvangeste. Epic logo, then black screen.

[Ryujinx_1.1.1050_2023-10-10_20-34-32.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/12863154/Ryujinx_1.1.1050_2023-10-10_20-34-32.log)

**Comment by Tvangeste** [comment]
With the latest **Ryujinx 1.1.1050** the game boots and starts just fine for me, I see some slowdown/stutter right after the logo, for a couple of seconds and after that the game plays smooth and stable for me.

I'm on nvidia 3080 Ti, multithreaded backend, vulkan, enabled shader cache, texture recompression, macro HLE, resolution scale 2x (1440p/2160p). VSync enabled.

**Comment by amurgshere** [comment]
This game works for me on MAC OS (M1 MacBook Air 16GB 8 GPU Cores) game version 1.5.2 (Vulcan Graphics), but I have to have it in handheld mode when it loads until it gets to the first splash screen, otherwise I just get black screen. Once it has loaded to splash screen I can switch it to docked mode and it keeps working. I get a solid 30 FPS from this game except for the shader compilation which calms down as more and more shaders are compiled. So I think the FPS issues people are talking about are just the shaders being compiled.

**Comment by Icsjoshj** [comment]
Ironically I think it runs better without applying any of the updates to the game.  Without applying updates it runs at 60fps instead  of 30fps.  Something else I noticed is without the updates you can also unlock the framerate and it varied between 85 to 110 fps on my ROG Ally and it seems to not effect the speed of the game (other than feeling smoother at the higher fps), but after applying the updates it will lock to 30fps regardless of being locked/unlocked.  The only issue I've noticed so far with not applying the updates is that after finishing races the car will run into obstacles constantly (just the AI driving after the race, so not affecting the player mid-race at all.
